### PR TITLE
feat(paddle frontend): Added ifft2 

### PR DESCRIPTION
**Pull request by teckno**
# PR Description

## Related Issue

Close #

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

File: ivy/functional/frontends/paddle/fft.py [hunk 1]
```diff
@@ -125,6 +125,16 @@ def ifft(x, n=None, axis=-1.0, norm="backward", name=None):
...
 
 
+@to_ivy_arrays_and_back
+def ifft2(x, s=None, axes=(-2, -1), norm="backward", name=None):
+    fft_result = ivy.fft2(x, s=s, dim=axes, norm=norm)
+    if norm == "forward":
+        fft_result /= ivy.sqrt(ivy.prod(ivy.shape(fft_result)))
+    elif norm == "ortho":
+        fft_result /= ivy.sqrt(ivy.prod(ivy.shape(x)))
+    return ivy.real(fft_result)
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/paddle/fft.py:135]: The return type should be of `complex` dtypes as specified in the docs

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 

Would you check the comments below

**Review by hmahmood24 — CHANGES_REQUESTED** [review]
Hey @teckno, can you please also link the issue that you are trying to resolving by commenting `Close #issue_number` ? Thanks!

**Comment by hmahmood24** [comment]
Closing this PR since it has been stale for more than 2 weeks now.

**Pull request closed without merging**